moveit_ros/benchmarks/src/simple_benchmarks/CombinePredefinedPosesBenchmark.cpp

```diff
@@ -59,11 +59,6 @@ class CombinePredefinedPosesBenchmark : public BenchmarkExecutor
     // Load planning scene
     if (!psm_)
       psm_.reset(new planning_scene_monitor::PlanningSceneMonitor("robot_description"));
-    if (!psm_->newPlanningSceneMessage(scene_msg))
-    {
-      ROS_ERROR_NAMED(LOGNAME, "Failed to load planning scene");
-      return false;
-    }
```

Comment:
I think the `newPlanningSceneMessage` tries to update the scene with an empty `scene_msg` thus the empty JointState error. This benchmark does not load a planing scene from ROS warehouse.

So the actual scene is loaded/created from the `psm_.reset` function.

Bottom line I think the error message was valid. It works as expected without spawning the JointState error.